### PR TITLE
Docker installation tutorial updates

### PR DESCRIPTION
**Pull request by ltagliaferri**
### What should this PR do?

Broke single tutorial into 3, added additional warnings regarding M1 and Windows installation workflows.

Deleted previous single-stream tutorial.

### Why are we making this change?

Clarify that the M1 chip on macOS is not supported by Sourcegraph. Also clarify the issue with Docker and Windows 10.

### What are the acceptance criteria? 

The language of the notes in the macOS and Windows tutorials are satisfactory to the Support team and address relevant concerns.

### How should this PR be tested?

Can check the Netlify preview and read the tutorials, or read the markdown files in the GitHub PR. 

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.

### CODE REVIEW

File: posts/how-to-install-sourcegraph-on-mac-with-docker.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+author: lisa-tagliaferri
+tags: [tutorial, Docker, installation, Sourcegraph]
+description: Learn how to install Sourcegraph on your Apple Mac with Docker
+image: /headers/sourcegraph-learn-header-6.svg
+imageAlt: Sourcegraph Learn
+socialImage: /headers/sourcegraph-learn-header-6.png
+type: posts
+---
+
+Sourcegraph is a tool that can allow you to search all the code that is important to you, from your own local repositories, to the software you build together with a team, to all the code that is available through open source projects. You can get started using Sourcegraph by using [Sourcegraph Cloud](https://sourcegraph.com/search) with your preferred web browser, or you may want to install Sourcegraph locally or on a cloud server or virtual machine.
```
Comment (Kelvin-Lee) [inline posts/how-to-install-sourcegraph-on-mac-with-docker.md:12]: Not sure if we want to go with "Sourcegraph Cloud" or "Sourcegraph cloud" (i.e. not sure if we want to capitalize Cloud or not). Another file uses the lowercase version.

As long as we're consistent I assume it's not highly consequential.

File: posts/how-to-install-sourcegraph-on-mac-with-docker.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+By installing Sourcegraph on your own hardware, you’ll be able to manage your own instance of Sourcegraph, providing you with complete control over who has access to your code. Your own installation will also allow you to connect your local (or server-side) code as well as any relevant Git servers. Git servers that you can connect to include GitHub, GitHub Enterprise, GitLab, Bitbucket, AWS CodeCommit, Perforce, and more.
+
+We’ll go through how to install Sourcegraph on your local macOS machine with Docker so that you can begin to use Sourcegraph on all of your code. 
+
+---
+**NOTE**:
```
Comment (Kelvin-Lee) [inline posts/how-to-install-sourcegraph-on-mac-with-docker.md:19]: This being included was my main request. Thanks Lisa!

**Review by Kelvin-Lee — APPROVED** [review]
One small inconsistency with capitalization of "Sourcegraph [C/c]loud". As long as it is consistent, I suspect it's not too important which we go with. Requesting that at least 1 be chosen, at Lisa's discretion.

Approving ahead of time for convenience.

File: posts/how-to-install-sourcegraph-on-mac-with-docker.md (hunk 1 above)
Comment (ltagliaferri) [reply posts/how-to-install-sourcegraph-on-mac-with-docker.md]: Good eye! I missed the changes, the lower-case is the version that the rest of the team has been discussing. Updated the other uses in the commit below.

**Pull request merged**